doc/ale-mercury.txt

```diff
@@ -0,0 +1,28 @@
+===============================================================================
+ALE Mercury Integration
+*ale-mercury-options*
```

Comment:
Put the documentation marker on the right margin, like in the other files. That might be why Travis was complaining here.